test-integration/scores.coffee

```diff
@@ -0,0 +1,127 @@
+{describe, CourseSelect, screenshot} = require './helpers'
+selenium = require 'selenium-webdriver'
+{expect} = require 'chai'
+
+TEACHER_USERNAME = 'teacher01'
+
+MAIN_ELEMENTS = [
+  '.scores-report',
+  '.course-scores-wrap',
+  '.nav-tabs',
+  '.export-button',
+  '.course-scores-title',
+  '.course-scores-container'
+]
+
+
+class StudentScores
+
+  constructor: (@test, @login, @courseTitle) ->
+    @test.login(@login)
+    CourseSelect.goTo(@test, @courseTitle)
+    @goToScores()
+
+  goToScores: ->
+    if @courseTitle is 'CC'
+      @test.waitClick(css: '.detailed-scores')
+    else
+      @test.waitClick(css: '.calendar-actions a:nth-child(3)')
+    @test.sleep(1500)
+
+  findMainElements: (elements) ->
+    for element in elements
+      e = @test.driver.findElement(css: element)
+      e.getAttribute('class').then (element) ->
+        console.log "found element: #{element}"
+      e.isDisplayed().then (element) ->
+        if not element
+          throw Error('element is not visible')
+
+  clickElement: (className, message) ->
+    @test.driver.findElement(css: className).click().then ->
+      console.log "clicked: #{message}"
```

Comment:
I thought it would be ok to log the different elements that are clicked.  I found it useful to watch as it runs through test.